Firebase/Core/FIRApp.m

```diff
@@ -613,11 +643,61 @@ - (NSString *)expectedBundleID {
 }
 
 // end App ID validation
-#pragma mark
+
+#pragma mark - Reading From Plist & User Defaults
+
+/**
+ * A wrapper to clear the data collection switch from the standard NSUserDefaults for easier testing
+ * and readability.
+ */
+- (void)clearDataCollectionSwitchFromUserDefaults {
+  NSString *key =
+      [NSString stringWithFormat:kFIRGlobalAppDataCollectionEnabledDefaultsKeyFormat, self.name];
+  [[NSUserDefaults standardUserDefaults] removeObjectForKey:key];
+}
+
+/**
+ * A wrapper to read the data collection switch from the standard NSUserDefaults for easier testing
```

Comment:
nit: Reads the data collection...
Please remove "A wrapper to" here and below

Reply:
Done.